extensions/wikia/Chat2/css/ChatWidget.scss

```diff
@@ -84,104 +86,19 @@ body.skin-oasis .ChatMonobookWidget {
 		right: 0;
 	}
 
-	.chat-whos-here {
-		@include clearfix;
-		border: 1px solid $color-page-border;
-		margin: 10px 0;
-		padding: 5px;
-		position: relative;
-
-		.arrow-left,
-		.arrow-right {
-			cursor: pointer;
-			line-height: 34px;
-			position: absolute;
-
-			&.disabled {
-				cursor: auto;
-				opacity: .5;
-			}
-		}
-
-		.arrow-left {
-			left: 0;
-			padding-left: 10px;
-
-			img {
-				@include arrow(6px, $color-chat-live, 'left');
-			}
-		}
-
-		.arrow-right {
-			right: 0;
-			padding-right: 10px;
-
-			img {
-				@include arrow(6px, $color-chat-live, 'right');
-			}
-		}
-
-		.carousel-container {
-			height: 34px;
-			left: 21px;
-			overflow: hidden;
-			position: relative;
-			width: 244px;
-			& > div {
-				position: relative;
-			}
-			ul {
-				@include clearfix;
-				list-style: none;
-				margin: 0;
-				position: absolute;
-			}
-			li {
-				float: left;
-				margin: 0 8px 0 0;
-			}
-		}
-		@if ($responsive-or-breakpoints-layout-enabled) {
-			.carousel-container {
-				left: 19px;
-				width: 218px;
-
-				li {
-					margin-right: 12px;
-				}
-			}
-		}
-
-		.avatar {
-			border: 1px solid $color-chat-avatar-border;
-			vertical-align: middle;
-		}
-	}
-
-	&.module h3 {
-		font-size: 13px;
-		padding: 0;
+	html[lang^='zh'] & .wds-is-squished {
+		font-size: 12px;
+		font-weight: 400;
+		padding: 9px 12px 8px;
 	}
 }
 
 // Note: a lot of this is borrowed from .WikiaRail .module
 .ChatWidget {
 	@include clearfix;
-	@include box-shadow(0px, 2px, 5px, darken($color-page,12%));
-	background: $color-page;
-	border: 1px solid $color-page-border;
-	clear: both;
-	display: inline-block;
-	@if lightness($color-page) < 50 {
-		@include box-shadow(0px, 2px, 5px, darken($color-page,7%));
-		background: $color-module-background;
-		border: 1px solid $color-page-border;
-	}
-	margin: 0 auto;
-	margin-bottom: 10px;
-	padding: 16px 8px 8px;
+	margin: 0 auto 30px;
 	position: relative;
-	width: 284px;
+	width: 300px;
```

Comment:
I remember that we were to put this value in a separate variable while working on clean-up ticket :)